data/snippets.ts

```diff
@@ -553,6 +553,13 @@ const coding = [
     keyword: "css-ac",
     type: "template",
   },
+  {
+    id: nanoid(),
+    name: "Make a New Folder and Open it in VS Code",
+    text: "mkdir {clipboard} && code -r {clipboard}",
+    keyword: "vscd-init",
+    type: "template",
+  },
```

Comment:
Nice! Can you make sure the properties are in the same order as all other snippets please?

Also can we shorten the title a little? Maybe "Create and Open Folder in VSCode"

Reply:
Thanks! I updated Title and order please check'em!